components/root/root.jsx

```diff
@@ -146,8 +147,74 @@ export default class Root extends React.Component {
         };
     }
 
+    getSidebarCategories = () => {
+        const sidebarCategories = [
+            document.getElementById('unreadsChannelList'),
+            document.getElementById('favoriteChannelList'),
+            document.getElementById('publicChannelList'),
+            document.getElementById('privateChannelList'),
+            document.getElementById('directChannelList'),
+        ].filter((element) => Boolean(element));
+
+        return sidebarCategories;
+    }
+
     handleAccessibilityKeys = (e) => {
-        if (isKeyPressed(e, Constants.KeyCodes.TAB)) {
+        if (isKeyPressed(e, Constants.KeyCodes.DOWN)) {
```

Comment:
Might be a good idea to split the contents of the if statement into a different functions since `handleAccessibilityKeys` is getting really long